src/tests/myFinalTests.spec.js

```diff
@@ -0,0 +1,34 @@
+// @ts-check
```

Comment:
Let's rename the file name, i.e. `sortingOnTheInventoryPage.spec.js`.
This title would be more appropriate.

Reply:
Updated